app/assets/javascripts/kuroko2/instance_linker.js

```diff
@@ -1,10 +1,25 @@
 jQuery(function ($) {
-  $('td.log').each(function(){
-    var logText = $(this).html();
-    $(this).html(
-      logText.replace(/instance#(\d+)\/(\d+)(?:\s+|\.)/, function(match, jobId, instanceId){
-	return '<a href="/definitions/' + jobId + '/instances/' + instanceId+ '">' + match + '</a>';
-      })
-    );
+  var MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;
+  if (!MutationObserver) {
+    return;
```

Comment:
PhantomJS supports MutationObserver from 2.0. But It seems that TravisCI uses PhantomJS 1.x currently (as default). So if browser does not support MutationObserver, this function (instance linker) should be disabled.